tests/functional/object/rgw/test_rgw_sts.py

```
@@ -186,6 +181,21 @@ def role_permission_policy(self):
 
+    def make_boto3_client(
+        self, service, endpoint, region, access_key, secret_key, session_token=None
+    ):
+        """Create a boto3 client with common RGW connection parameters."""
+        kwargs = dict(
+            endpoint_url=endpoint,
+            aws_access_key_id=access_key,
+            aws_secret_access_key=secret_key,
+            region_name=region,
+            verify=False,
+        )
```

Ruff (0.15.18)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)